src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj

```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{0febe054-68ac-446f-b999-9068736d3cec}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AssemblyName>System.Private.Uri.Tests</AssemblyName>
-    <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
```

Comment:
Are all the tests in here new and only useful for netstandard1.7 stuff? If so I'm good with this change if not then we need to correctly split the tests.